spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/SimpleMessageListenerContainer.java

```diff
@@ -633,9 +633,15 @@ private void considerAddingAConsumer() {
 		}
 	}
 
+	private int getConsumersThatHaveNotInitiatedStopping() {
+		synchronized (this.consumersMonitor) {
+			return (int) this.consumers.values().stream().filter(x -> x = Boolean.TRUE).count();
```

Comment:
Please, move your comment to the top level. Now it isn't visible because of your changes.